AfsprakenRegels.md

```diff
@@ -781,6 +781,8 @@ Er kan sprake zijn van 1 naam en/of definitie die voor meerdere modelelementen g
 
 Echter, voor bijvoorbeeld voor een relatiesoort of een attribuutsoort kan het ook nodig zijn om dezelfde naam en/of definitie te specificeren. Dit is mogelijk, maar het is dan niet geheel duidelijk of er hetzelfde bedoeld wordt, of dat de (exacte) overeenkomt in een informatiemodel (zoals bv. in een UML informatiemodel) een toevalligheid is. Daarom gelden de volgende afspraken: 
 
+* Als de **uri** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Gebied met attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie" en een objecttype Boom met een attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie".
+
 * Als de **definitie** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het dezelfde definitie is. Bijvoorbeeld: attribuutsoort overboeking met _definitie_ "Het bedrag in euro's" en attribuutsoort koopsom met _definitie_ "Het bedrag in euro's".
 
 * Als de **naam** _en_ de **definitie** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Huis met attribuutsoort oppervlakte: : "De oppervlakte in hele vierkante meters" en objecttype Appartement met attribuutsoort oppervlakte : "De oppervlakte in hele vierkante meters".
```

Comment:
Is hier duidelijk genoeg wat kenmerk is? Want het interessante is dat we in deze drie punten een verschil maken in identiteit:
- Bij URI's gaat het om de identiteit van een Modelelement (los van LD, maar feitelijk voor de identiteit van een modelelement in MIM ZELF
- Bij Naam+definitie gaat het om de identiteit van een kenmerk (een LD dingetje: predicates, objects, etc)